src/CommunityToolkit.Maui.UnitTests/Behaviors/AnimationBehavior_Tests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using CommunityToolkit.Maui.Behaviors;
+using CommunityToolkit.Maui.UnitTests.Mocks;
+using Microsoft.Maui.Controls;
+using System.Linq;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Behaviors;
+
+public class AnimationBehavior_Tests : BaseTest
+{
+	[Fact]
+	public void TabGestureRecognizerAttachedWhenNoEventSpecified()
+	{
+		var boxView = new BoxView();
+		boxView.Behaviors.Add(new AnimationBehavior());
+		var gestureRecognizers = boxView.GestureRecognizers.ToList();
+		
+		Assert.Single(gestureRecognizers);
+		Assert.IsType<TapGestureRecognizer>(gestureRecognizers[0]);
+	}
+
+	[Fact]
+	public void TabGestureRecognizerNotAttachedWhenEventSpecified()
+	{
+		var boxView = new BoxView();
+		boxView.Behaviors.Add(new AnimationBehavior()
+		{
+			EventName = nameof(BoxView.Focused),
+		});
+		var gestureRecognizers = boxView.GestureRecognizers.ToList();
+
+		Assert.Empty(gestureRecognizers);
```

Comment:
I suggest using FluentAssertion. The assertion looks more natural and readable:
`gestureReconginzers.Should().BeEmpty();`

Reply:
😮 I didn't notice FluentAssertions was available in the project, love it! I will update the assertions.